### ISSUE TITLE
Upgrade error: not a valid zip file

### ISSUE THREAD

**Issue by tonirug** [syncthing/syncthing#4158]
Hello,
a W10 device in my cluster is having problems in upgrading to v0.14.28
The error message is: **main.go:560: FATAL: Upgrade: zip: not a valid zip file**
Attached is the output of **syncthing -upgrade** executed with **STTRACE=upgrade** set
[upgrade.txt](https://github.com/syncthing/syncthing/files/1013886/upgrade.txt)

It is not a new issue. I am having it since a few months also on another W10 box when upgrading via the GUI button. In that case it was still possible to upgrade via the command line, so not a big issue. Now same problem from the command line. 
The zip file, if downloaded from the URL, is ok.

Best
Toni

### Version Information

Syncthing Version: v0.14.27
OS Version: Windows 10
Browser Version: (if applicable, for GUI issues)

**Comment by AudriusButkevicius** [comment]
As the issue template explains, please use the [forum](https://forum.syncthing.net/) for support.
Please use the search on the forum and the closed issue tracker for simillar issues before starting a new thread. 
If this issue is later proven to be a bug, we will reopen this issue or open a new one.

**Comment by calmh** [comment]
99% of weird unexplainable failures on Windows are caused by antivirus. This smells exactly like that.
